cfme/tests/infrastructure/test_vm_rest.py

```diff
@@ -135,3 +138,44 @@ def test_delete_vm_from_collection(vm):
         casecomponent: Infra
     """
     delete_resources_from_collection([vm], not_found=True, num_sec=300, delay=10)
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1684681])
+@pytest.mark.provider(
+    classes=[InfraProvider],
+    selector=ONE,
+    override=True,
+    required_fields=[["cap_and_util", "capandu_vm"]],
+)
+def test_filtering_vm_with_multiple_ips(appliance, provider):
+    """
+    Polarion:
+        assignee: pvala
+        caseimportance: high
+        casecomponent: Rest
+        initialEstimate: 1/4h
+        setup:
+            1. Add a provider.
+        testSteps:
+            1. Select a VM with multiple IP addresses and note one ipaddress.
+            2. Send a GET request with the noted ipaddress.
+                GET /api/vms?expand=resources&attributes=ipaddresses&filter[]=ipaddresses=':ipaddr'
+        expectedResults:
+            1.
+            2. Selected VM must be present in the resources sent by response.
+
+    Bugzilla:
+        1684681
+    """
+    # 1
+    vm = appliance.collections.infra_vms.instantiate(
+        provider.data["cap_and_util"]["capandu_vm"], provider
+    )
+    # 2
+    result = appliance.rest_api.collections.vms.filter(
```

Comment:
@valaparthvi 
- Don't you think this is confusing(```# 1, # 2``` - In my understanding, you are referencing steps or setup)?
- According to me, It will be better if you add ```testSteps``` as  **inline** comments and you can remove test steps and expected results from polarion data.
Please let me know your view.

Reply:
@ganeshhubale I think it does indeed reference to `testSteps` but I don't think she needs to remove anything from Polarion docblock as that is part of what we use for uploading tests to polarion.

Reply:
Hey @ganeshhubale I don't think it's that confusing, it seems fine to me. There are only 2 teststeps, I think I would rather keep this as is.